src/main/java/org/jpeek/App.java

```diff
@@ -230,6 +231,14 @@ public void analyze() throws IOException {
                 )
             );
         }
+        if (this.params.containsKey("PCC")) {
```

Comment:
@llorllale I don't like this ``if`` here, it is very procedural... but the whole design of Jpeek is 100% procedural. I think we need to report a bug and ask for refactoring, usage of objects!